relay-general/src/protocol/event.rs

```diff
@@ -343,7 +343,7 @@ pub struct Event {
     /// can be the git SHA for the given project, or a product identifier with a semantic version.
     #[metastructure(
         max_chars = "tag_value",  // release ends in tag
-        deny_chars = "\r\n\x0c\t/\\",
+        // release allowed chars are validated in the sentry-release-parser crate!
```

Comment:
this change does mean that the release name is only validated in the store normalizer. I think we will need to follow up with moving this part of store normalization to run before metrics extraction as well

Reply:
Does it even make sense to remove the `deny_chars` at this point? Or should we do that as part of the follow up?

Reply:
A separate PR that moves this in one go would've been slightly more semantic, but let's keep it this way in light of moving fast.

Reply:
https://getsentry.atlassian.net/browse/INGEST-1285